apps/brukerdialog-sanity/schemaTypes/inntektsjustering/fellesKomponenterSchemaType.ts

```diff
@@ -35,6 +35,16 @@ export const fellesKomponenterSchemaType = defineType({
                 }),
             ],
         }),
+        defineField({
+            name: 'skjemaTittel',
+            title: 'Skjema tittel',
+            type: 'object',
+            fields: [
```

Comment:
Det er ikke mulig å flytte dette ut for lettere å gjenbruke det, samt at man slipper å endre 100 steder hvis det er noe som må fikses?
```
[ defineField({ name: 'NB', type: 'string' }),
defineField({ name: 'NN', type: 'string' }),
defineField({ name: 'EN', type: 'string' }), ]
```

Reply:
Skjønner ikke helt hva du mener 😅 Skjema tittelen vil være lik over hele skjemaet. Altså det vil alltid stå `Meld fra om inntekt til neste år`